rust/connlib/gateway/src/main.rs

```diff
@@ -67,7 +67,7 @@ fn main() -> Result<()> {
     let url = parse_env_var::<Url>(URL_ENV_VAR)?;
     let secret = parse_env_var::<String>(SECRET_ENV_VAR)?;
     let device_id = get_device_id();
-    let mut session = Session::connect(url, secret, device_id, CallbackHandler).unwrap();
+    let mut session = Session::connect(url, secret, device_id, None, CallbackHandler).unwrap();
```

Comment:
Why does the gateway not have a logger? Then you could get rid of the `Option`.

Reply:
The gateway does have a logger, just not a file logger -- only STDOUT/ERR. See [this comment](https://github.com/firezone/firezone/pull/1973#issuecomment-1705376117) for the reasons why I think this is justified.

Reply:
Fair :)